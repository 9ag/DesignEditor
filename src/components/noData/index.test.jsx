
import React from 'react';
import renderer from 'react-test-renderer';

import NoData from './index';


describe("NoData component test", () => {
  it("NoData snapshot test", () => {
    const component = renderer.create(<NoData />).toJSON();
    expect(component).toMatchSnapshot();
  });

});