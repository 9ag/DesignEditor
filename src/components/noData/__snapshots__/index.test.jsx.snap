// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoData component test NoData snapshot test 1`] = `
<div
  className="no-data-content"
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div
    className="no-data"
  >
    <img
      src="../Sources/images/pegman/no-dashboard.png"
    />
    <h4>
      No result found
    </h4>
    <p>
      Please try to search again.
    </p>
  </div>
</div>
`;
